plugins/woocommerce/includes/class-wc-brands.php

```diff
@@ -396,15 +396,28 @@ public function brand_description() {
 	public function show_brand() {
 		global $post;
 
-		if ( is_singular( 'product' ) ) {
+		$has_block_template = apply_filters( 'woocommerce_has_block_template', false, 'taxonomy-product_brand' ); // phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingHookComment
```

Comment:
Have you considered the loading order between where this filter is [originally implemented](https://github.com/woocommerce/woocommerce/blob/07c289399d2faabf8ebda0124bda3c26b6e5ac7e/plugins/woocommerce/includes/class-wc-template-loader.php#L171), WC Brands usage here, and how that impacts existing usages of `add_filter( 'woocommerce_has_block_template' )`?